schedule/yast/skip_registration/skip_registration_pvm.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
it is really strange we do this only for power, so I would just remove test coverage for all, you can remove this file too, so we don't execute those steps in any architecture, for non-registration we still have another test suites.